up-l1/zenoh.adoc

```diff
@@ -178,3 +183,91 @@ Encoding::WithSuffix(
 ----
 
 * Zenoh takes data as a buffer simply and sends it directly. Ignore the shared memory (reference) temporarily.
+
+=== Work Flow
+
+The section provides the details how it works in different message types.
+
+==== Transfer Publish message
+
+1. A uEntity (`origin-authority/origin-entity`) creates a message with
+    * source: `up://origin-authority/origin-entity/origin-version/origin-resource.instance#message`
+    * sink: `-`
+
+2. `UTransport.send` maps source to Zenoh key
+    * local (w/o origin-authority): `upl/` + `origin-resource-id + origin-entity-id + origin-version`
+    * remote (with origin-authority): `upr/` + `origin-authority-id/` + `origin-resource-id + origin-entity-id + origin-version`
+
+3. Subscriber registers a listener for
+    * `up://origin-authority/origin-entity/origin-version/origin-resource.instance#message`
+
+4. which `UTransport.registerListener` maps to Zenoh key
+    * local (w/o origin-authority): `upl/` + `origin-resource-id + origin-entity-id + origin-version`
+    * remote (with origin-authority): `upr/` + `origin-authority-id/` + `origin-resource-id + origin-entity-id + origin-version`
+
+5. Since this is **Publish URI**, `Zenoh subscriber` will be used to bind the `registerListener`
+
+==== Transfer Notification message
+
+1. A uEntity (`origin-authority/origin-entity`) creates a message with
+    * source: `up://origin-authority/origin-entity/origin-version/origin-resource.instance`
+    * sink: `up://dest-authority/dest-entity/dest-version/dest-resource.instance`
+
+2. `UTransport.send` maps sink to Zenoh key
+    * local (w/o dest-authority): `upl/` + `dest-resource-id + dest-entity-id + dest-version`
+    * remote (with dest-authority): `upr/` + `dest-authority-id/` + `dest-resource-id + dest-entity-id + dest-version`
+
+3. Receiver (dest-authority/dest-entity) registers a listener for
+    * `up://dest-authority/dest-entity/dest-version/dest-resource.instance`
+
+4. which `UTransport.registerListener` maps to Zenoh key
```

Comment:
see comment above.

Reply:
Michael identified a key issue now on the client side in registerListener() that I'll document in #81